hw/top_darjeeling/BUILD

```diff
@@ -33,7 +33,7 @@ sim_dv(
     linker_script = "//sw/device/lib/testing/test_framework:ottf_ld_silicon_creator_slot_a",
     otp_mmap = "//hw/top_darjeeling/data/otp:otp_ctrl_mmap.hjson",
     rom_scramble_config = "//hw/top_darjeeling/data/autogen:top_darjeeling.secrets.dev.gen.hjson",
-    seed_cfg = "//hw/top_darjeeling/data:top_darjeeling_seed.dev.hjson",
+    secret_top_cfg = "//hw/top_darjeeling/data/autogen:top_darjeeling.secrets.dev.gen.hjson",
```

Comment:
Can this be overridden with a flag/argument to Bazel?

Reply:
This is the follow-up from the first PR. At the moment it's coded to the dev secrests/seed. But that needs to be overwritten in a future PR. In this PR we only switch from passing the seed to passing the secrets that contain the seeds (and more).